lib/pages/friendslist_page.dart

```diff
@@ -0,0 +1,223 @@
+import 'package:breview/widgets/FriendListWidget.dart';
+import 'package:flutter/material.dart';
+import 'package:breview/widgets/FriendsWidget.dart';
+import 'package:breview/util/Constants.dart';
+import 'package:breview/services/crud.dart';
+
+class FriendsList extends StatefulWidget {
+  FriendsList({Key key}) : super(key: key);
+
+  @override
+  _FriendsListState createState() => _FriendsListState();
+}
+
+class _FriendsListState extends State<FriendsList> {
+  TextEditingController textController;
```

Comment:
give an appropriate naming for this controller

Reply:
Done